rules/integrations/kubernetes/credential_access_kubernetes_secrets.toml

```diff
@@ -0,0 +1,54 @@
+[metadata]
+creation_date = "2022/05/17"
+maturity = "production"
+updated_date = "2022/05/17"
+integration = "kubernetes"
+
+[rule]
+author = ["Elastic"]
+description = """
+This rule detects a user attempting to access either one or all Kubernetes secrets within a given namespace, through a Kubernetes API server request. Kubernetes secrets often hold sensitive information, including credentials. Actions taken to gather credentials to container and cloud infrastructure can allow for privilege escalation within Kubernetes and access to external systems. 
+"""
+false_positives = [
+    """
+    An administrator may need to access secrets for a legitimate purpose.
+    """,
+]
+index = ["filebeat-*", "logs-kubernetes.*"]
+language = "kuery"
+license = "Elastic License v2"
+name = "Access Kubernetes Secrets"
```

Comment:
```suggestion
name = "Kubernetes Secrets Accessed"
```

May recommend making it more past-tense for the user. Looking at the alert may saying "Accessed" may give some motivation to review.